### ISSUE TITLE
Newer docker support

### ISSUE THREAD

**Issue by rallisf1** [litespeedtech/cyberpanel#567]
## envrironment
* cyberpanel 2.0.3
* centos 7

## how to reproduce
upgrade docker following guide on https://docs.docker.com/engine/install/centos/

## the problem relevance
Docker Manager is pretty useless for me as most images (I care about) on the hub require at least version 16, 1.13 is pretty ancient.

## what I expect
* install.sh to include official docker repo and latest versions of docker-engine & docker-compose
* Docker Manager inside CyberPanel to be compatible with latest docker

P.S. I will try 2.1.1 when I get the time, although I didn't see any commits regarding the matter.

**Comment by rallisf1** [comment]
Seems the problem is limited to Centos. On ubuntu version 19 of docker is installed and works beautifully through cyberpanel's management gui.

I remember reading on the cyberpanel documentation that docker management is only available on Ubuntu, perhaps this is why, but then tried deploying a couple containers on Centos 7 and it worked until I stumbled upon an image that didn't work and the reason was the old docker version. Still, installing the official docker repo and latest version, as I did, should not break cyberpanel's docker functionality.

So until this is fixed, everyone who wants to run docker images should stick with Ubuntu.
